src/lib/positions/getLiquidation.ts

```diff
@@ -0,0 +1,65 @@
+import { BigNumber } from "ethers";
+import {
+  BASIS_POINTS_DIVISOR,
+  getLiquidationPriceFromDelta,
+  LIQUIDATION_FEE,
+  MARGIN_FEE_BASIS_POINTS,
+  MAX_LEVERAGE,
+} from "../legacy";
+
+type GetLiquidationParams = {
+  size: BigNumber;
+  collateral: BigNumber;
+  averagePrice: BigNumber;
+  isLong: boolean;
+  fundingFee?: BigNumber;
+};
+
+function calculateTotalFees(size: BigNumber, fundingFees: BigNumber): BigNumber {
```

Comment:
i thinkg calculateTotalFees is not related exactly for liquidation and can be used in other places?

Maybe rename the file just to positions/utils?

Reply:
it factors in `LIQUIDATION_FEE` so i guess it's okay to have it here?